android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/DictionaryTest.java

```diff
@@ -3,34 +3,35 @@
 import org.junit.Test;
 
 import java.util.ArrayList;
-import java.util.HashMap;
+import java.util.TreeMap;
 import java.util.List;
 import java.util.Locale;
 import java.util.Map;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
 public class DictionaryTest extends BaseTest {
     @Test
     public void testCreateDictionary() {
         Dictionary address = new Dictionary();
         assertEquals(0, address.count());
-        assertEquals(new HashMap<String, Object>(), address.toMap());
+        assertEquals(new TreeMap<String, Object>(), address.toMap());
```

Comment:
I will switch it back to HashMap. And will consider another solution for map comparison.